prow/config.yaml

```diff
@@ -841,21 +836,16 @@ presubmits:
     rerun_command: "/test pull-security-kubernetes-e2e-gce-gci"
     trigger: "(?m)^/test pull-security-kubernetes-e2e-gce-gci,?(\\s+|$)"
   - name: pull-security-kubernetes-e2e-gce-gpu
-    agent: jenkins
-    always_run: false
-    context: pull-security-kubernetes-e2e-gce-gpu
-    rerun_command: "/test pull-security-kubernetes-e2e-gce-gpu"
-    trigger: "(?m)^/test pull-security-kubernetes-e2e-gce-gpu,?(\\s+|$)"
-  - name: pull-security-kubernetes-e2e-gce-gpu-prow
     agent: kubernetes
     skip_branches:
     - release-1.4
     - release-1.5
     - release-1.6
-    always_run: false
-    context: pull-security-kubernetes-e2e-gce-gpu-prow
-    rerun_command: "/test pull-security-kubernetes-e2e-gce-gpu-prow"
-    trigger: "(?m)^/test pull-security-kubernetes-e2e-gce-gpu-prow,?(\\s+|$)"
+    always_run: true
```

Comment:
Can we skip reporting so batch jobs know to skip this?

Reply:
I'm fine with this, @mindprince?

Reply:
I think we should skip reporting and after we see a good signal we can revisit if we should make it blocking etc. I will update the PR.

Reply:
Added `skip_report: true` below.